manifests/params.pp

```diff
@@ -441,7 +441,7 @@
   $server_idle_timeout                    = 1200000
   $server_web_idle_timeout                = 30000
   $server_connect_timeout                 = 120000
-  $server_enable_ruby_profiler            = false
+  $server_enable_ruby_profiler            = true
```

Comment:
To clarify, you're suggesting to drop `$server_enable_ruby_profiler` and use `$server_puppetserver_metrics` for both?